policyengine_us/parameters/gov/states/ny/tax/income/credits/ctc/additional/amount.yaml

```
@@ -0,0 +1,32 @@
+description: New York State matches this fraction of the Empire State Credit as an additional credit, based on the federal adjusted gross income.
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: /1
+  period: year
+  label: New York additional Empire State Child Credit amount
```

```suggestion
  label: New York additional Empire State Child Credit match
```
As I think value indicates a dollar value